### ISSUE TITLE
Compare Old to New Datapoint

### ISSUE THREAD

**Issue by dave4jr** [soynatan/django-easy-audit#26]
Hello! Fantastic library, thanks so much for your work on this! Quick question for you...

Is there a way to see what the old values were on the UPDATE events? So basically when I go into the admin and click on django easy audit CRUD, it shows me the updated value, but doesn't show me what the updated value was before it was updated. Is there a way for me to add that? THanks so much!!

Dave

**Comment by soynatan** [comment]
Hello, Dave. Currently it's not possible to do that. On issue #24 we discuss the possibility of logging both the full updated value, and in another field, only the values that changed since the last update. But we have no plans to log what the value was before the update.

If you want to do that, you need to tweak some things. If you know about signals, you should change the file "signals.py", and make it use the signal `pre_save` instead of the current `post_save`. That way, before saving you could access the original object with a query, and then compare that object with the instance being saved. Check out [this SO thread](https://stackoverflow.com/questions/5582410/django-how-to-access-original-unmodified-instance-in-post-save-signal).

Hope it helps. Thank you!

**Comment by dave4jr** [comment]
Thank you so much for taking the time to write back all that, and it actually really did help. I'm almost done and thing I've got it working for the most part, but was only able to throw an hour or so at it.

Just for some more background, I built a piece of software that helps manage makerspaces. So check-in / check-out members, calendar, scheduling, inventory, built an entire POS system in Django and lots lots more. Currently we have about 200 users on the software and I use your application to monitor activity within the admin so in that sense it's pretty critical to know what the value was before that so you know if that person was the one that changed it or was it changed before that and they just updated it, etc... So again, thank you so much for your help and again, Great app!

Dave
